src/client/datascience/variablesView/types.ts

```diff
@@ -1,17 +1,31 @@
-import { CssMessages, IGetCssRequest, IGetCssResponse, SharedMessages } from '../messages';
-import { IVSCWebviewViewProvider } from '../types';
+import { IVariableExplorerHeight } from '../../../datascience-ui/interactive-common/redux/reducers/types';
+import {
+    InteractiveWindowMessages,
+    IShowDataViewer
+} from '../../datascience/interactive-common/interactiveWindowTypes';
+import { CssMessages, IGetCssRequest, IGetCssResponse, IGetMonacoThemeRequest, SharedMessages } from '../messages';
+import { IGetMonacoThemeResponse } from '../monacoMessages';
+import { IJupyterVariablesRequest, IJupyterVariablesResponse, IVSCWebviewViewProvider } from '../types';
 
-export namespace VariableViewMessages {
-    export const Started = SharedMessages.Started;
-    export const UpdateSettings = SharedMessages.UpdateSettings;
-}
-
-// Map all messages to specific payloads
-export class IVariableViewMapping {
-    public [VariableViewMessages.Started]: never | undefined;
-    public [VariableViewMessages.UpdateSettings]: string;
+// Mapping of Message to payload that our VariableViewPanel needs to support
+export class IVariableViewPanelMapping {
+    public [InteractiveWindowMessages.ForceVariableRefresh]: never | undefined;
+    public [InteractiveWindowMessages.SettingsUpdated]: string;
+    public [InteractiveWindowMessages.Activate]: never | undefined;
+    public [InteractiveWindowMessages.ShowDataViewer]: IShowDataViewer;
+    public [InteractiveWindowMessages.GetVariablesRequest]: IJupyterVariablesRequest;
+    public [InteractiveWindowMessages.GetVariablesResponse]: IJupyterVariablesResponse;
+    public [InteractiveWindowMessages.VariableExplorerToggle]: boolean;
+    public [InteractiveWindowMessages.SetVariableExplorerHeight]: IVariableExplorerHeight;
+    public [InteractiveWindowMessages.VariableExplorerHeightResponse]: IVariableExplorerHeight;
     public [CssMessages.GetCssRequest]: IGetCssRequest;
     public [CssMessages.GetCssResponse]: IGetCssResponse;
+    public [CssMessages.GetMonacoThemeRequest]: IGetMonacoThemeRequest; // Needed for started message
+    public [CssMessages.GetMonacoThemeResponse]: IGetMonacoThemeResponse;
```

Comment:
Why do we need monaco theme in variable view?
We dont' have a monaco editor there, hence this doesnb't make sense.

Reply:
If not required, i think we should remove this

Reply:
I think that it was in as it was part of the interactive startup commands and I wanted to have as little custom code as possible between the variable view and interactive / old notebook views. But that seems lazy, pretty sure I can cut this out and it will still work fine, might have to make a new action, but that's easy enough. Will try.

Reply:
Chopped out the editor specific bits of startup.